src/plot_api/plot_api.js

```diff
@@ -326,6 +326,9 @@ Plotly.plot = function(gd, data, layout, config) {
     // so that the caller doesn't care which route we took
     return Promise.all(gd._promises).then(function() {
         return gd;
+    }, function() {
+        // clear the promise queue if one of them got rejected
+        gd._promises = [];
```

Comment:
This feels dangerous and like it will lead to impossible to debug situations... Should we not _at least_ log a warning?